src/OrchardCore/OrchardCore.ContentManagement.Abstractions/VersionOptions.cs

```diff
@@ -0,0 +1,35 @@
+namespace OrchardCore.ContentManagement;
+
+public class VersionOptions
+{
+    /// <summary>
+    /// Gets the latest version.
+    /// </summary>
+    public static VersionOptions Latest { get { return new VersionOptions { IsLatest = true }; } }
+
+    /// <summary>
+    /// Gets the latest published version.
+    /// </summary>
+    public static VersionOptions Published { get { return new VersionOptions { IsPublished = true }; } }
+
+    /// <summary>
+    /// Gets the latest draft version.
+    /// </summary>
+    public static VersionOptions Draft { get { return new VersionOptions { IsDraft = true }; } }
+
+    /// <summary>
+    /// Gets the latest version and creates a new version draft based on it.
+    /// </summary>
+    public static VersionOptions DraftRequired { get { return new VersionOptions { IsDraft = true, IsDraftRequired = true }; } }
+
+    /// <summary>
+    /// Gets all versions.
+    /// </summary>
+    public static VersionOptions AllVersions { get { return new VersionOptions { IsAllVersions = true }; } }
```

Comment:
Isn't this supposed to go away?

Reply:
isn't it better to just support it instead? if someone really wants all versions, they can now.

Reply:
We said it was ambiguous, and didn't cope with a list on ids. WE should just add a custom method to get all the versions ,if it doesn't already exists (queryign content item versions)

Reply:
okay. I removed the AllVersions and instead added `Task<IEnumerable<ContentItem>> GetAllVersionsAsync(string contentItemId)` to the content manager.

Reply:
Good!